packages/cfd/src/Containers/dynamic-leverage/dynamic-leverage-market-card.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { Table, Text } from '@deriv/components';
+import { localize } from '@deriv/translations';
+import { TDynamicLeverageMarketCardProps } from 'Containers/props.types';
+
+export const DynamicLeverageMarketCard = ({
+    market,
+    leverage,
+    market_example,
+    dynamicLeverages,
+}: TDynamicLeverageMarketCardProps) => (
+    <div className='dynamic-leverage-modal__market'>
+        <div className='dynamic-leverage-modal__market-title'>
+            <Text data-testid='market_title' size='xs' weight='bolder' align='center'>
+                {localize(market)}
```

Comment:
These data are already localized from dynamic-leverage-content file. Do we need localize here?

Reply:
I don't think we need to localize again. Updated.